cmd/format-coverage.go

```diff
@@ -20,11 +20,12 @@ import (
 )
 
 type CoverageFormatter struct {
-	In        formatters.Formatter
-	InputType string
-	Output    string
-	Prefix    string
-	writer    io.Writer
+	CoveragePath string
```

Comment:
yes, exactly. Actually if we don't `CoveragePath` and `InputType`, it will run the inference logic